[2019-04-01 20:29:45,184] {/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py:1761} ERROR - Exception on /tentor [GET]
Traceback (most recent call last):
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 104, in wrapper
    return fn(*args, **kwargs)
  File "/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/tentor/resources.py", line 173, in get
    if satuan == mil:
NameError: name 'mil' is not defined
[2019-04-01 20:29:45,187] {/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/__init__.py:40} WARNING - REQUEST_LOG	{"request": {"jarak": "5"}, "response": {"message": "Internal Server Error"}} GET
[2019-04-01 20:29:56,243] {/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py:1761} ERROR - Exception on /tentor [GET]
Traceback (most recent call last):
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 104, in wrapper
    return fn(*args, **kwargs)
  File "/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/tentor/resources.py", line 173, in get
    if satuan == mil:
NameError: name 'mil' is not defined
[2019-04-01 20:29:56,247] {/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/__init__.py:40} WARNING - REQUEST_LOG	{"request": {"jarak": "5"}, "response": {"message": "Internal Server Error"}} GET
[2019-04-01 20:30:06,816] {/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/__init__.py:40} WARNING - REQUEST_LOG	{"request": {"jarak": "5"}, "response": "tes"} GET
[2019-04-01 20:30:22,564] {/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/__init__.py:40} WARNING - REQUEST_LOG	{"request": {"jarak": "5"}, "response": "ft"} GET
[2019-04-01 20:30:45,252] {/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py:1761} ERROR - Exception on /tentor [GET]
Traceback (most recent call last):
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 104, in wrapper
    return fn(*args, **kwargs)
  File "/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/tentor/resources.py", line 173, in get
    if satuan == mi:
NameError: name 'mi' is not defined
[2019-04-01 20:30:45,254] {/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/__init__.py:40} WARNING - REQUEST_LOG	{"request": {"jarak": "5"}, "response": {"message": "Internal Server Error"}} GET
[2019-04-01 20:31:23,589] {/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py:1761} ERROR - Exception on /tentor [GET]
Traceback (most recent call last):
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask/views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alphatech/.local/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 104, in wrapper
    return fn(*args, **kwargs)
  File "/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/tentor/resources.py", line 173, in get
    if satuan == mi:
NameError: name 'mi' is not defined
[2019-04-01 20:31:23,593] {/home/alphatech/Documents/Tugas_ATA_Batch2_Aswin/Last_Final/Last_Week/bimbel-buddies/blueprints/__init__.py:40} WARNING - REQUEST_LOG	{"request": {"jarak": "5"}, "response": {"message": "Internal Server Error"}} GET
